source/github-helpers/index.ts

```diff
@@ -138,6 +138,11 @@ export const cacheByRepo = (): string => getRepo()!.nameWithOwner;
 // Commit lists for files and folders lack a branch selector
 export const isRepoCommitListRoot = (): boolean => pageDetect.isRepoCommitList() && document.title.startsWith('Commits');
 
+export const isUrlReachable = mem(async (url: string): Promise<boolean> => {
```

Comment:
```suggestion
export const isUrlUnreachable = mem(async (url: string): Promise<boolean> => {
```

Reply:
🥹

Reply:
It looks like I made a couple of mistakes when renaming:

- I meant to change the 404 check to just use `response.ok`
- I forgot to flip every condition to actually match "is reachable", to avoid double negations